### ISSUE TITLE
Code Sandbox Errors when using OUFR as Implicit Dependency

### ISSUE THREAD

**Issue by JasonGore** [OfficeDev/office-ui-fabric-react#8656]
### Environment Information 
 
- __Package version(s)__: 6.164.8 (@uifabric/experiments 6.70.0)
- __Browser and OS versions__: n/a

### Please provide a reproduction of the bug in a codepen:
 

 https://codesandbox.io/s/8nqy90rl92

#### Actual behavior:
 
Code Sandbox fails to use @uifabric/experiments (and OUFR package implicitly as a dependency of @uifabric/experiments) starting around v6.67.0 with this error:

![image](https://user-images.githubusercontent.com/26070760/55824756-62876200-5ab9-11e9-8104-46a7bf264a11.png)

Further investigation notes a bunch of 403 errors when trying to load:

![image](https://user-images.githubusercontent.com/26070760/55824808-792db900-5ab9-11e9-97df-f9fe661e220b.png)

And loading any of these errors seems to indicate some bundle size issue:

![image](https://user-images.githubusercontent.com/26070760/55824835-877bd500-5ab9-11e9-920c-f6fa2d02d925.png)

 
#### Expected behavior:
 
Can write Code Sandbox examples using @uifabric/experiments.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

**Comment by joschect** [comment]
@JasonGore I can confirm this is a bug, I'll add this to our bugs and take a look at it.

**Comment by ecraig12345** [comment]
As a temporary-ish workaround, maybe we could make `office-ui-fabric-react-min` which only includes whichever output type is needed by codesandbox (and excludes `**/examples`, `**/*.doc.*`, and `src`)? Downside is we'd have to make a similar `@uifabric/experiments-min` that depends on min OUFR...

**Comment by ecraig12345** [comment]
Another idea is make `office-ui-fabric-react-doc` which includes the actual output plus examples, doc pages, and src, then make normal OUFR exclude all those. (Would be safer to do as of 7, but could be done within 6 probably, since it would be weird if anyone external was directly referencing doc pages and examples.)

**Comment by JasonGore** [comment]
I'm just going to add that @khmakoto had another recent experiments CodeSandbox that ran fine so the one I posted may have a legitimate code issue not related to bundle size. It's possible that this bundle size error has been occurring for some time and does not actually prevent CodeSandbox samples from running correctly.

**Comment by micahgodbolt** [comment]
Is this still a bug?

**Comment by JasonGore** [comment]
There's a codesandbox provided. I think it'd be pretty easy to rev the version numbers and still see if the problem exists.

**Comment by micahgodbolt** [comment]
Updated sandbox with more current versions. Still repro'ing

https://codesandbox.io/s/dependency-error-8656-1w7ui

**Comment by JasonGore** [comment]
I was doing some sanity checking by simplifying the example and also trying it in CodePen and found the issue. `SplitMenuButton` does not exist in experiments. I think it was renamed to `SplitButton` at some point and broke this example, but [changing the name works](https://codesandbox.io/s/dependency-error-8656-evk7l).

The fact that typing doesn't work at all in Code Sandbox isn't helping things. It would have more clearly identified this component as not existing. I made a [Code Sandbox issue](https://github.com/codesandbox/codesandbox-client/issues/1647) for this some time ago but it's still open.

Closing this issue for now.
